Doc/library/asyncio-eventloop.rst

```diff
@@ -81,8 +86,6 @@ This documentation page contains the following sections:
   loop APIs.
 
 
-.. _asyncio-event-loop:
-
```

Comment:
I moved this reference to the top of the page primarily because it seemed more appropriate for the "event loops" link in https://docs.python.org/3.9/library/asyncio.html.

Reply:
Reverted to the original ref location, as requested by Yury.